Sources/ElementaryHxSSE/HTMLAttribute+HxSSE.swift

```diff
@@ -0,0 +1,29 @@
+import Elementary
+import ElementaryHTMX
+
+public extension HTMLAttribute.hx where Tag: HTMLTrait.Attributes.Global {
+    /// A namespace for the HTMX SSE-extension.
+    /// See the [htmx-sse reference](https://github.com/bigskysoftware/htmx-extensions/blob/main/src/sse/README.md) for more information.
+    enum sse {}
+}
```

Comment:
in order to keep the spelling close to plain HTML I prefer to have the `.sse` enum on the HTMLAttribute directly.
this way we can spell it as `div(.hx.ext(.sse), .sse.connect("my-url")) {}` and stick to the general pattern of elementary.